packages/cli/generators/relation/templates/controller-relation-template-belongs-to.ts.ejs

```diff
@@ -0,0 +1,34 @@
+import {
+  repository,
+} from '@loopback/repository';
+import {
+  param,
+  get,
+} from '@loopback/rest';
+import { <%= sourceModelClassName %>, <%= targetModelClassName %> } from '../models';
+import { <%= sourceRepositoryClassName %> } from '../repositories';
+
+export class <%= controllerClassName %> {
+  constructor(
+    @repository(<%= sourceRepositoryClassName %>)
+    public <%= paramSourceRepository %>: <%= sourceRepositoryClassName %>,
+  ) { }
+
+  @get('/<%= sourceModelName %>s/{id}/<%= targetModelName %>', {
+    responses: {
+      '200': {
+        description: "<%= targetModelClassName %> belonging to <%= sourceModelClassName %>",
```

Comment:
We use single quotes, not double quotes.

```suggestion
        description: '<%= targetModelClassName %> belonging to <%= sourceModelClassName %>',
```

Reply:
Fixed